_data/simple-icons.json

```
@@ -7059,6 +7059,11 @@
             "hex": "00AFAA",
             "source": "https://www.here.com/company/media-assets"
         },
+        {
+            "title": "Hermes",
+            "hex": "0091CD",
+            "source": "https://www.myhermes.de"
```

```suggestion
            "source": "https://www.myhermes.de/assets/touchicons/favicon.svg"
```
We can link directly to the file here as it isn't immediately apparent what the source is.